spec.bs

```diff
@@ -3811,9 +3869,12 @@ request header</h3>
 The \`<dfn http-header><code>Sec-Ad-Auction-Fetch</code></dfn>\` request header is an optional
 [=structured header=] with of type [=structured header/boolean=]. [:Sec-Ad-Auction-Fetch:] will only
 be set on a [=request=] whose [=request/initiator type=] is `"fetch"`, made with the
-{{RequestInit/adAuctionHeaders}} option set to `true`. If [:Sec-Ad-Auction-Fetch:] is equal to `?1`,
+{{RequestInit/adAuctionHeaders}} option set to `true` or on an
+<a href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#navigate-an-iframe-or-frame">iframe navigation</a>
+request with the <{iframe/adAuctionHeaders}> <a spec=html>content attribute</a>

```

Comment:
All of the instances of `<{iframe/adAuctionHeaders}>` should be updated to not be camel case, per the above review comment about casing.

Reply:
Done. Thanks again!